testcharms/charm-hub/charms/actions/README.md

```diff
@@ -0,0 +1,3 @@
+# Actions charm
+
+A charm used in the actions integration tests. I has an action called `fortune`, see charmcraft.yaml for more.
```

Comment:
```suggestion
A charm used in the actions integration tests. I have an action called `fortune`, see charmcraft.yaml for more.
```

(Unless this was lol speak on purpose).

Reply:
The new official Canonical writing guidelines state British English is to be replaced by lol speak where possible.

I've changed it to "It has" :)